nix/nixpkgs.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I recommend to add a Makefile target to update this file.

Reply:
Fixed